libraries/core/src/main/java/com/paritytrading/philadelphia/FIXMessageParser.java

```diff
@@ -25,6 +25,8 @@
  */
 public class FIXMessageParser {
```

Comment:
Considering that we cannot customize the message parser (there is no constructor argument and it is `new`-ed in the constructor), it would be better off if this class was inlined in `FIXConnection`.

Not a hard opinion.